### PR TITLE
[Docs]Update video tutorial to use VideoCallParticipantView

### PR DESCRIPTION
**Pull request by ipavlidakis**
### 🔗 Issue Links

_Provide all JIRA tickets and/or GitHub issues related to this PR, if applicable._

### 🎯 Goal

_Describe why we are making this change._

### 📝 Summary

_Provide bullet points with the most important changes in the codebase._

### 🛠 Implementation

_Provide a detailed description of the implementation and explain your decisions if you find them relevant._

### 🎨 Showcase

_Add relevant screenshots and/or videos/gifs to easily see what this PR changes, if applicable._

|  Before  |  After  |
| -------- | ------- |
|  `img`   |  `img`  |

### 🧪 Manual Testing Notes

_Explain how this change can be tested manually, if applicable._

### ☑️ Contributor Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] This change follows zero ⚠️ policy (required)
- [x] This change should receive manual QA
- [ ] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (Docusaurus, tutorial, CMS)

### 🎁 Meme

_Provide a funny gif or image that relates to your work on this pull request. (Optional)_

### CODE REVIEW

File: docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx [hunk 1]
```diff
@@ -226,16 +245,25 @@ struct FloatingParticipantView: View {
...
     var body: some View {
         if let participant = participant {
             VStack {
                 HStack {
                     Spacer()
 
-                    VideoRendererView(id: participant.id, size: size) { videoRenderer in
-                        videoRenderer.handleViewRendering(for: participant, onTrackSizeUpdate: { _, _ in })
-                    }
+                    VideoCallParticipantView(
```
Comment (martinmitrevski) [inline docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx:256]: Do we need the logic for the floating view? This participant is not in the scroll list, so maybe we can leave the previous way as an example.

File: docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx [hunk 2]
```diff
@@ -183,6 +183,13 @@ var body: some Scene {
         }
     }
 }
+
+private func changeTrackVisibility(_ participant: CallParticipant?, isVisible: Bool) {
```
Comment (martinmitrevski) [inline docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx:187]: We should have some text explanation why we do this.

**Review by martinmitrevski — COMMENTED** [review]
Looks good, just few small things before merging.

File: docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx [hunk 3]
```diff
@@ -253,32 +280,35 @@ Now when you run the app, you'll see your local video in a floating video elemen
...
+.onDisappear{ onChangeTrackVisibility(participant, false) }
 ```
 
 It only displays the video and doesn't add any other UI elements. The video is lazily loaded, and only requested from the video infrastructure if you're actually displaying it. So if you have a video call with 200 participants, and you show only 10 of them, you'll only receive video for 10 participants. This is how software like Zoom and Google Meet make large calls work.
 
-`FloatingParticipantVideo` renders a display of your own video.
 
 ```swift
-FloatingParticipantView(participant: call.state.me)
+FloatingParticipantView(participant: call.state.me, onChangeTrackVisibility: changeTrackVisibility(_:isVisible:))
```
Comment (martinmitrevski) [inline docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx:303]: Do we still need this?

File: docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx [hunk 4]
```diff
@@ -253,32 +280,35 @@ Now when you run the app, you'll see your local video in a floating video elemen
 
 Let's review the changes we made.
 
-[VideoRendererView](../../ui-components/video-renderer/) is one of our primary low-level components.
+We added the `changeTrackVisibility` in our app and we propagate its call to the other subviews we create. When this method is being called we are asking the Call object to make the participant's track visible or not visible. This is important when the view goes off-screen (e.g. when the user backgrounds the app) to reduce energy and data consumption.
```
Comment (martinmitrevski) [inline docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx:283]: "e.g. when the user backgrounds the app" - better example would be when the user goes off-screen while scrolling through participants during a call. Pausing of tracks in the background is handled differently (listening to app lifecycle notifications over view lifecycle).

File: docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx [hunk 5]
```diff
@@ -226,6 +249,7 @@ struct FloatingParticipantView: View {
 
     var participant: CallParticipant?
     var size: CGSize = .init(width: 120, height: 120)
+    var onChangeTrackVisibility: (CallParticipant?, Bool) -> Void
```
Comment (martinmitrevski) [inline docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx:252]: We don't need this here, right? (the local video doesn't go out of screen while scrolling)

**Review by martinmitrevski — COMMENTED** [review]
Few more nits and we are ready

File: docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx (hunk 5 above)
Comment (ipavlidakis) [reply docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx:252]: 👍

**Review by martinmitrevski — APPROVED** [review]
✅

**Pull request merged**